src/Commands/Load.php

```diff
@@ -30,7 +30,13 @@ public function handle()
             return;
         }
 
-        $name = $this->argument('name') ?: $this->askForSnapshotName();
+        $use_latest_snapshot = $this->option('latest') ?: false;
+
+        if ($use_latest_snapshot) {
+            $name = $snapShots->last()->name;
```

Comment:
What happens when there are no snapshots? Add a test for that case as well.

Reply:
That is already caught by the first check in the command

Reply:
Right 🙂